e2e/cypress/integration/integrations/custom_slash_commands/custom_slash_commands_spec.js

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+import {addNewCommand} from './slash_commands_spec';
+
+describe('Slash commands', () => {
+    const trigger = 'my_trigger';
+    let user1;
+    let user2;
+    let team1;
+    let commandURL;
+
+    before(() => {
+        cy.requireWebhookServer();
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            team1 = team;
+
+            cy.apiGetChannelByName(team.name, 'town-square').then(({channel}) => {
+                commandURL = `${Cypress.env().webhookBaseUrl}/send_message_to_channel?channel_id=${channel.id}`;
+            });
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T696 Can\'t delete other user\'s slash command', () => {
+        cy.apiAdminLogin(user1);
+
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // # Create new Slash command
+        addNewCommand(team1, trigger, 'http://dot.com');
+
+        cy.visit('/admin_console/user_management/permissions/system_scheme');
+
+        // # Click manage slash commands checkbox and confirm
+        cy.findByTestId('all_users-integrations-manage_slash_commands-checkbox').then((el) => {
+            if (!el.hasClass('checked')) {
+                el.click();
+            }
+        });
+        cy.get('#saveSetting').click();
```

Comment:
This can be simplified by using the these util functions:
https://github.com/larkox/mattermost-webapp/blob/906ac1d7a84520fab13f471a739d50d68fa9019d/e2e/cypress/integration/enterprise/oauth/oauth_spec.js#L50-L52